src/relative-time-element.ts

```diff
@@ -382,6 +383,10 @@ export class RelativeTimeElement extends HTMLElement implements Intl.DateTimeFor
     this.setAttribute('format-style', value)
   }
 
+  get noTitle(): boolean {
+    return this.hasAttribute('no-title') && this.getAttribute('no-title') === 'true'
```

Comment:
Checking for `'true'` isn’t super idiomatic. I’d prefer us to check just for the presence of the attribute, using hasAttribute (so no getAttribute call). Then the markup can be `<relative-time no-title>`. I agree a setter might be useful here too, especially for React

Reply:
Ok that makes more sense! So we would just pass in `no-title` with no value?

Reply:
Yup!